admin/class-create-block-theme-admin.php

```diff
@@ -327,6 +374,11 @@ function add_templates_to_zip( $zip, $export_type ) {
 			}
 
 			$template->content = _remove_theme_attribute_in_block_template_content( $template->content );
+
+			//NOTE: Dashes are replaced with \u002d in the content that we get (noteably in things like css variables used in templates)
+			// This replaces that with dashes again.
+			$template->content = str_replace( '\u002d', '-', $template->content );
```

Comment:
this looks like the original content was encoded, is it maybe [this](https://github.com/WordPress/gutenberg/blob/7aa042605ff42bb437e650c39132c0aa8eb4ef95/lib/global-styles.php#L28) what's happening? Maybe we can use the decode function instead in case there's more than dashes that we should consider?

Reply:
Ah, very good call.  Thank you.  That felt pretty shabby when I wrote it. :P

Reply:
I tried very hard to do this. :(
I can't figure out how. 

AFAICT this is something that should be addressed in Gutenberg and until it is perhaps this "good enough" solution will be ok?

Reply:
I mentioned it to @scruffian on his latest PR to the export functionality in GB, hopefully there will be a fix for it.

Reply:
I think this actually happens when the template is modified using the site editor...